functests/1_performance/cpu_management.go

```diff
@@ -72,7 +74,7 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			Expect(capacityCPU - allocatableCPU).To(Equal(int64(len(listReservedCPU))))
 		})
 
-		It("[test_id:28026][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify CPU affinity mask, CPU reservation and CPU isolation on worker node", func() {
+		It("[test_id:27081][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify CPU affinity mask, CPU reservation and CPU isolation on worker node", func() {
```

Comment:
Question so I can understand: why is the test id changed?

Reply:
Test_id is linked to test case in polarion. Here we had just wrong test_id. 
We may have another reasons for changing test_id, for example when we are updating test to new logic - we may have to create new test case, so test_id will be changed.

Reply:
This is also better come as  a separate patch in the same PR. @MarSik do we have clear guidelines when to split patches or is OK as it is (I don't want @dshchedr to do more work if not necessary).

Reply:
A short explanation in the commit message is enough I think. It fixes the same test if I see right.